app/clients/lider_client.rb

```diff
@@ -0,0 +1,47 @@
+class LiderClient < ChromeClient
+    BASE_URL = 'https://www.lider.cl/supermercado/search?Ntt='
+  
+    def products_by_query(query: nil)
+      ensuring_browser_closure do
+        go_to_products_page(query)
+        
+        scraped_products = []
+        browser.search('.product-details').wait(:present, timeout: 5.0).map do |product|
+          scraped_products.push({
+            brand: get_brand(product),
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[Correctable] Layout/FirstHashElementIndentation: Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.